File: C++/1036.cpp
#include <stdio.h>
#include <math.h>

 
int main() {
 double A,B,C;
 scanf("%lf %lf %lf",&A,&B,&C);
 if((B*B-(4*A*C))>=0 && A!=0){
 	printf("R1 = %.5lf\n", ((-B + sqrt((B*B)-(4.0*A*C)))/(2.0*A)));
 	printf("R2 = %.5lf\n", ((-B - sqrt((B*B)-(4.0*A*C)))/(2.0*A)));
 }
 else printf("Impossivel calcular\n");
 return 0;
 		
   
}

File: Python/1036.py
import math                              
                                         
a, b, c= input().split()                 
a = float(a)                             
b = float(b)                             
c = float(c)                             
                                         
delta = (b*b) - (4*a*c)                  
if a<=0:                                 
    print('Impossivel calcular')         
elif delta<0:                            
    print('Impossivel calcular')         
else:                                    
    r1 = (-b + math.sqrt(delta))/(2*a)   
    r2 = (-b - math.sqrt(delta))/(2*a)   
    print(f'R1 = {r1:.5f}')              
    print(f'R2 = {r2:.5f}')   